lib/shopify_api/webhooks/registry.rb

```diff
@@ -193,7 +193,16 @@ def process(request)
             raise Errors::NoWebhookHandler, "No webhook handler found for topic: #{request.topic}."
           end
 
-          handler.handle(topic: request.topic, shop: request.shop, body: request.parsed_body)
+          if handler.use_handle_webhook?
+            handler.handle_webhook({ topic: request.topic, shop: request.shop, body: request.parsed_body,
+              webhook_id: request.webhook_id, api_version: request.api_version, })
+          else
+            handler.handle(topic: request.topic, shop: request.shop, body: request.parsed_body)
+            ShopifyAPI::Logger.warn(<<~WARNING)
+              DEPRECATED: Use ShopifyAPI::Webhooks::Handler#handle_webhook
+              instead of ShopifyAPI::Webhooks::Handler#handle
```

Comment:
Can we add a link to the great docs you added here to help folks understand why we did it?